go/teams/service_helper.go

```diff
@@ -1613,7 +1613,7 @@ func CanUserPerform(ctx context.Context, g *libkb.GlobalContext, teamname string
 
 	ret.DeleteChannel = admin
 	ret.RenameChannel = writer
-	ret.EditChannelDescription = writer
+	ret.EditChannelDescription = admin
```

Comment:
Do we want this change?

Reply:
Description and avatar are the same?

Reply:
Definitely not.

Reply:
so someone with write permission is supposed to be able to edit the team description? It didn't work that way yesterday, so I assumed they were equivalent.

Reply:
Assuming this is used for how it is named, this is referring to channel description.

Reply:
:white_check_mark: